src/main/java/bio/terra/common/stairway/StairwayProperties.java

```diff
@@ -87,4 +100,22 @@ public Duration getCompletedFlightRetention() {
   public void setCompletedFlightRetention(Duration completedFlightRetention) {
     this.completedFlightRetention = completedFlightRetention;
   }
+
+  public String getGcpPubSubTopicId() {
+    return gcpPubSubTopicId;
+  }
+
+  public StairwayProperties gcpPubSubTopicId(String gcpPubSubTopicId) {
+    this.gcpPubSubTopicId = gcpPubSubTopicId;
+    return this;
```

Comment:
Do you need these fluent-style methods for all the properties?

Reply:
No. I fixed them to match the other accessors.